### ISSUE TITLE
seeing message "[mason-lspconfig.nvim] Server "sumneko_lua" is not a valid entry in ensure_installed" when launching neovim

### ISSUE THREAD

**Issue by ajaydwarkani** [nvim-lua/kickstart.nvim#179]
I am seeing the below message every time I start the nvim

```
[mason-lspconfig.nvim] Server "sumneko_lua" is not a valid entry in ensure_installed. Make sure to only provide lspconfi
g server names.
```

![image](https://user-images.githubusercontent.com/4305382/219450815-fa0682ed-fd2f-48bd-a27a-e544cca5285f.png)

**To Reproduce**
1. Install neovim via brew `brew install newvim`
2. Copied `kickstart.nvim/init.lua` to `~/.config/nvim/init.lua`
3. Launched nvim
4. After all the packages were installed, I quit and relaunched the nvim
5. I see the above stated message
6. I hit enter to ignore the message
7. nvim installs some LSPs
8. quit and restarted the nvim, I again see the same message.

**Desktop (please complete the following information):**
- OS: Mac OS Ventura 13.1
- Terminal: zsh 5.8.1 

** Neovim Version **
```
NVIM v0.8.3
Build type: Release
LuaJIT 2.1.0-beta3
Compiled by brew@Ventura

Features: +acl +iconv +tui
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/usr/local/Cellar/neovim/0.8.3/share/nvim"
```

**Comment by feoh** [comment]
Work is onging to fix this in #178

**Comment by lashomb** [comment]
also seeing this error, same os and shell versions

**Comment by LudoPinelli** [comment]
Already fixed for the next PR as @feoh said.
As a workaround until then, you just have to replace `sumneko_lua` by `lua_ls` at line 346.

**Comment by justinfarrelldev** [comment]
> Already fixed for the next PR as @feoh said. As a workaround until then, you just have to replace `sumneko_lua` by `lua_ls` at line 346.

Doing this fixed it for me, thank you!

**Comment by Josee9988** [comment]
Same here in Ubuntu 22.04

**Comment by tjdevries** [comment]
Fixed in latest by #178 (but there are quite a few changes in the PR)

**Comment by definetly-not-carl** [comment]
> Already fixed for the next PR as @feoh said. As a workaround until then, you just have to replace `sumneko_lua` by `lua_ls` at line 346.

I had the same issue in Fedora37, but this solved it
